multimedia/streamlink/Portfile

```
@@ -4,11 +4,11 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                streamlink
-version             1.7.0
+version             6.5.1
 revision            0
-checksums           rmd160  da6be6fdc2dc905ec2682f2eb7d1455659a3ec9a \
-                    sha256  f87a62a47212d94769929bd040d9c186b461643bdbda06f839b99ec9efefb87a \
-                    size    728957
+checksums           rmd160  47679c55b5776d87ffdb803ddd00ff0218c390c6 \
+                    sha256  207fb4ce99c35bfeb1b8f7c76b96cfcb4076ad6881c61eaea553c2ec13d97c57 \
+                    size    717297
 
 categories-append   multimedia net
```

Lint warning: `Error: Portfile parent directory multimedia does not match primary category python`

Seems like we'd want to change this to the following to fix:

```suggestion
categories          multimedia net
```

---

Thanks, fixed.